### PR TITLE
Feature/pie chart

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

File: src/BsRecharts__PieChart.re [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+[@bs.module "recharts"]
+external reactClass : ReasonReact.reactClass = "PieChart";
+
+[@bs.obj]
+external makeProps :
+  (
+    ~height: int=?,
+    ~margin: margin=?,
+    ~onClick: (Js.t({..}), ReactEventRe.Mouse.t) => unit=?,
+    ~onMouseEnter: (Js.t({..}), ReactEventRe.Mouse.t) => unit=?,
```
Comment (liubko) [inline src/BsRecharts__PieChart.re]: I think the first argument is nullable, here and in other handlers. Can you please check this and update bindings if needed?

**Comment by liubko** [comment]
@medson10 thanks for a PR 👍 

my only concern is the first argument in event handlers..

File: src/BsRecharts__PieChart.re (hunk 1 above)
Comment (medson10) [reply src/BsRecharts__PieChart.re]: yes they are indeed, https://github.com/recharts/recharts/blob/3b807be36f35e0ae8ecb6918cd57f6c0bc257f4a/src/chart/generateCategoricalChart.js#L495 all them use this function getMouseInfo, except the onMouseLeave the first argument is not nullable in this one https://github.com/recharts/recharts/blob/3b807be36f35e0ae8ecb6918cd57f6c0bc257f4a/src/chart/generateCategoricalChart.js#L1029

**Comment by medson10** [comment]
@liubko I updated all them, thanks for the review, congratulations guys you are doing a very good job with these bindings

**Comment by liubko** [comment]
@medson10 cool 👍 

another thing I forgot to mention, would you mind to add a simple example of a pie chart to `examples/`? can be done separately from this PR

**Comment by medson10** [comment]
@liubko sure, I can do it, I'll send this other PR soon

**Pull request merged**